src/Actions/DB/ExplainAction.php

```diff
@@ -1,42 +1,12 @@
 <?php
-/**
- * @link http://www.yiiframework.com/
- * @copyright Copyright (c) 2008 Yii Software LLC
- * @license http://www.yiiframework.com/license/
- */
-
 namespace Yiisoft\Yii\Debug\Actions\DB;
 
-use yii\base\Action;
-use yii\web\HttpException;
 use Yiisoft\Yii\Debug\Panels\DbPanel;
 
-/**
- * ExplainAction provides EXPLAIN information for SQL queries
- *
- * @property \Yiisoft\Yii\Debug\Controllers\DefaultController|\yii\web\Controller|\Yiisoft\Yii\Console\Controller $controller the controller that owns this action
- *
- * @author Laszlo <github@lvlconsultancy.nl>
- * @since 2.0.6
- */
-class ExplainAction extends Action
+class ExplainAction
```

Comment:
I think the description of the purpose of the class was helpful.

Reply:
Ok I will restore it